actors/builtin/miner/miner_state.go

```diff
@@ -109,6 +109,9 @@ type MinerInfo struct {
 	// This is computed from the proof type and represented here redundantly.
 	WindowPoStPartitionSectors uint64
 
+	// The maximum number of partitions allowed per Post deadline.
+	MaxPartitionsPerDeadline uint64
```

Comment:
This is a global. Is there any reason to include it in the miner state?

Reply:
Have moved this to the abi package.

Reply:
But why are we storing this in the miner state as well? This is a global limit, not a per-miner limit.